doc/installationguide.md

```diff
@@ -211,6 +226,10 @@ used for the same purpose. For instance:
     the IoTAgent runs in a single thread. For more details about multi-core functionality, please refer to the
     [Cluster](https://nodejs.org/api/cluster.html) module in Node.js and
     [this section](howto.md#iot-agent-in-multi-thread-mode) of the library documentation.
+-  **fallbackTenant** - For Linked Data Context Brokers which do not support multi-tenancy, this provides an alternative mechanism for suppling the `NGSILD-Tenant` header.
+    Note that for backwards compatibility with NGSI v2, the `fiware-service` header is already used as alternative if the `NGSILD-Tenant` header is not supplied.
+-  **fallbackPath** - For Linked Data Context Brokers which do not support a service path, this provides an alternative mechanism for suppling the `NGSILD-Path` header.
+    Note that for backwards compatibility with NGSI v2, the `fiware-service-path` header is already used as alternative if the `NGSILD-Path` header is not supplied.
```

Comment:
In addition, the right header for the path is `fiware-servicepath`. The `fiware-service-path` should be fixed in this documenation (and code, if this is more than a documentation typo and the PR code is not using the right one).

Reply:
This is a typo - please delete the hyphen. I am awaiting the final definition of Multitenancy support within NGSI-LD, so currently all four of   `NGSILD-Tenant` and  `NGSILD-Path` and  `fiware-service` and `fiware-servicepath` are sent.

Reply:
```suggestion
    Note that for backwards compatibility with NGSI v2, the `fiware-servicepath` header is already used as alternative if the `NGSILD-Path` header is not supplied. Note that NGSILD-Path has not been yet included in the NGSI-LD standard (it has been proposed for the v1.3, but no decission has been done yet), take into account it could change
```

What do you think?

Reply:
Upgraded:

```suggestion
    Note that for backwards compatibility with NGSI v2, the `fiware-servicepath` header is already used as alternative if the `NGSILD-Path` header is not supplied. Note that NGSILD-Path has not yet been included in the NGSI-LD standard (it has been proposed for the next update of the standard, but the final decision has yet been confirmed), take into account it could change
```

Reply:
Fixed in 42294e3